A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -85,6 +85,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 	        }
         this.electricity = this.querySelector("#Electricity");
         this.climbTransitionGroundAltitude = null;
+		this.currentFlightPhase = FlightPhase.FLIGHT_PHASE_PREFLIGHT;
```

Comment:
tabs

Reply:
idk why it looks like that on git. was normal in my file before uploading.